templates/slim/helpers.rb

```diff
@@ -3,14 +3,13 @@
 require 'asciidoctor'
```

Comment:
This does not work in Asciidoctor.js because Opal will try to load this module from the `OPAL_PATH`.

Reply:
Is that version check even necessary given that the way to use this module is through bundler?

Reply:
It’s not necessary, I suggest to remove it.